docs/fastdds/library_overview/includes/functionalities.rst

```diff
@@ -0,0 +1,92 @@
+Functionalities
+---------------
+
+Fast DDS has some added features that can be implemented and configured by the user in his/her application.
+These are outlined below.
+
+Discovery Protocols
+^^^^^^^^^^^^^^^^^^^
+
+The discovery protocols define the mechanisms by which DataWriters publishing under a given Topic, and DataReaders
+subscribing to that same Topic are matched, so that they can start sharing data.
+This applies at any point in the communication process.
+Fast DDS provides the following discovery mechanisms:
+
+* **Simple Discovery**.
+  This is the default mechanism.
```

Comment:
I'd mention that it's defined in a standard and provides compatibility with other DDS implementations